web_widget_child_selector/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+Use child_selection widget on a field.
```

Comment:
typo error that comes before the migration. The widget is child_selector. Can you change it please :smile:

Reply:
Of course! 😉